tests/tests.js

```diff
@@ -450,7 +450,7 @@ exports.defineAutoTests = function () {
                         expect(transfer.onprogress).toHaveBeenCalled();
 
                         // check that there is no file
-                        root.getFile(localFilePath, null, unexpectedCallbacks.fileSystemWin, done);
+                        root.getFile(fileName, null, unexpectedCallbacks.fileSystemWin, done);
```

Comment:
That sounds like a bug in `getFile`. The test code should obey what the docs say, and the implementation should be fixed, no? This sounds like an actual bug that the tests are uncovering.